.pre-commit-config.yaml

```diff
@@ -36,8 +36,3 @@ repos:
       language: system
       entry: "bash tools/sort_scalers.sh"
       files: .*scale_handler\.go$
-    - id: revive
-      language: system
-      name: Run revive against the code
-      entry: make revive
-      files: .*\.go$
```

Comment:
Would you mind adding a replacement command that will use golangci so everyone can run it locally?

Reply:
https://github.com/ultraware/whitespace

Whitespace is a linter that checks for unnecessary newlines at the start and end of functions, if, for, etc.

Reply:
fixed